common/static/coffee/src/discussion/views/new_post_view.coffee

```diff
@@ -87,7 +87,6 @@ if Backbone?
               url: url
               type: "POST"
               dataType: 'json'
-              async: false # TODO when the rest of the stuff below is made to work properly..
```

Comment:
Reviewers: I removed this because sinon doesn't have a good way to handle synchronous requests, and also because I can't see how it can be a good idea for the UX. We should think through what if any implications there might be for this change.